pkg/grizzly/server.go

```diff
@@ -289,7 +289,7 @@ func (s *Server) updateWatchedResource(name string) error {
 		}
 		resources, err = s.ParseBytes(b)
 	} else {
-		resources, err = s.ParseResources(s.ResourcePath)
+		resources, err = s.ParseResources(name)
```

Comment:
This is a drive-by fix: let's parse the resource that changed instead of re-parsing all of them.